### PR TITLE
Move bib976 to Instance

### PR DESCRIPTION
**Pull request by klngwll**
- [x] I have runt integTest

If [LXL-3304](https://jira.kb.se/browse/LXL-3304) moves bib976 to Instance, this needs to be merged.

### CODE REVIEW

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 1]
```diff
@@ -20484,14 +20484,39 @@
...
       "i2": {"property": "marc:bib976-i2", "marcDefault": " "},
       "$a": {"property": "marc:bib976-a"},
-      "$b": {"property": "marc:bib976-b"}
+      "$b": {"property": "marc:bib976-b"},
+      "_spec": [
+        {
+          "source": {
+            "976": {
+              "subfields": [
+                {"a": "a"},
```
Comment (lrosenstrom) [inline whelk-core/src/main/resources/ext/marcframe.json:20499]: The only 967's that are moved to the instance are those that are missing 976$a but maybe that's irrelevant here.

**Review by lrosenstrom — APPROVED** [review]
LGTM!

File: whelk-core/src/main/resources/ext/marcframe.json (hunk 1 above)
Comment (klngwll) [reply whelk-core/src/main/resources/ext/marcframe.json:20499]: If the others are becoming Classification objects yes :)

**Review by sarwik — APPROVED** [review]
LGTM!

**Pull request merged**